src/main/java/de/doubleslash/keeptime/view/ReportController.java

```diff
@@ -263,7 +263,8 @@ private void updateReport(final LocalDate dateToShow) {
          }
 
          final HBox projectButtonBox = new HBox();
-         projectButtonBox.getChildren().add(createCopyProjectButton(onlyCurrentProjectWork));
+         projectButtonBox.getChildren().add(createCopyProjectNotesButton(onlyCurrentProjectWork));
+         projectButtonBox.getChildren().add(createCopyProjectNameButton(onlyCurrentProjectWork.get(0).getProject().getName()));
```

Comment:
I'd say using the project name directly is easier and also more readable. What do you think?
```suggestion
         projectButtonBox.getChildren().add(createCopyProjectNameButton(project.getName()));
```